### DISCUSSION TITLE
issues with nodes

### DISCUSSION THREAD

**Discussion by patrickburnier** [freeplane/freeplane#1858] [Help]
Hi, 

complete newbie here, trying to get a grasp with this really nice software. I have created a number of nodes from  the core in the middle, but when moving some of them around, others are moving too. How do I stop this to happen? 
Many thanks in advance for your help!

**Comment by abc16361** [comment]
Right click on the node and set  'Free positioned node' on the right-click menu, then you can move the node freely without affecting other nodes.
<img width="449" alt="202406030754" src="https://github.com/freeplane/freeplane/assets/101362122/c324a065-072c-4e0d-afd9-ccd2855ee1c5">
